<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>report on tanaike</title>
    <link>https://tanaikech.github.io/tags/report/</link>
    <description>Recent content in report on tanaike</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>ja</language>
    <copyright>&amp;copy; 2017. All rights reserved.</copyright>
    <lastBuildDate>Sat, 11 Jun 2022 09:52:39 +0900</lastBuildDate><atom:link href="https://tanaikech.github.io/tags/report/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Report: Recent Value of ScriptApp.getService().getUrl() in Google Apps Script</title>
      <link>https://tanaikech.github.io/2022/06/11/report-recent-value-of-scriptapp.getservice.geturl-in-google-apps-script/</link>
      <pubDate>Sat, 11 Jun 2022 09:52:39 +0900</pubDate>
      
      <guid>https://tanaikech.github.io/2022/06/11/report-recent-value-of-scriptapp.getservice.geturl-in-google-apps-script/</guid>
      <description>There is a method of ScriptApp.getService().getUrl() for obtaining the Web Apps URL in Google Apps Script.
  Before the V8 runtime is released, this method had returned the endpoint like https://script.google.com/macros/s/{deploymentId}/exec.
  After the V8 runtime was released, the endpoint of https://script.google.com/macros/s/{deploymentId}/dev was returned.
  Now, it seems that this returns https://script.google.com/macros/s/###/exec.
  But, in the current stage, when I access this endpoint, the message of Sorry, unable to open the file at this time.</description>
    </item>
    
  </channel>
</rss>
